diff --git a/third_party/webrtc/p2p/base/port.cc b/third_party/webrtc/p2p/base/port.cc
index e26461dbe4..4f6a378d80 100644
--- a/third_party/webrtc/p2p/base/port.cc
+++ b/third_party/webrtc/p2p/base/port.cc
@@ -204,6 +204,11 @@
     return nullptr;
 }
 
+bool Port::IsValidStunMappedAddress(const SocketAddress& address) {
+  return (address.family() == AF_INET || address.family() == AF_INET6) &&
+         !address.IsUnresolvedIP() && !address.IsAnyIP() && address.port() > 0;
+}
+
 void Port::AddAddress(const SocketAddress& address,
                       const SocketAddress& base_address,
                       const SocketAddress& related_address,
