diff --git a/third_party/blink/renderer/modules/webgl/webgl_rendering_context_base.cc b/third_party/blink/renderer/modules/webgl/webgl_rendering_context_base.cc
index 9e3748ea3b3e..8fb3c741c692 100644
--- a/third_party/blink/renderer/modules/webgl/webgl_rendering_context_base.cc
+++ b/third_party/blink/renderer/modules/webgl/webgl_rendering_context_base.cc
@@ -4566,6 +4566,10 @@
       return ScriptValue::CreateNull(script_state->GetIsolate());
     case WebGLDebugRendererInfo::kUnmaskedRendererWebgl:
       if (ExtensionEnabled(kWebGLDebugRendererInfoName)) {
+        if (!ungoogled::CurrentPersona().webgl_real) {
+          return WebGLAny(script_state,
+                          String(ungoogled::CurrentPersona().webgl_renderer));
+        }
         if (ungoogled::CurrentPersona().webgl_real)
           return WebGLAny(script_state,
                           String(ContextGL()->GetString(GL_RENDERER)));
@@ -4595,6 +4599,10 @@
       return ScriptValue::CreateNull(script_state->GetIsolate());
     case WebGLDebugRendererInfo::kUnmaskedVendorWebgl:
       if (ExtensionEnabled(kWebGLDebugRendererInfoName)) {
+        if (!ungoogled::CurrentPersona().webgl_real) {
+          return WebGLAny(script_state,
+                          String(ungoogled::CurrentPersona().webgl_vendor));
+        }
         if (ungoogled::CurrentPersona().webgl_real)
           return WebGLAny(script_state,
                           String(ContextGL()->GetString(GL_VENDOR)));
